typespec-tests/tsp/enum.tsp

```diff
@@ -71,9 +72,10 @@ interface EnumOp {
   @route("/color")
   getColor(): Color;
 
-  @get
-  @route("/colormodel")
-  getColorModel(): ColorModel;
+  // TODO: the response as Union is ambiguous
+  // @get
+  // @route("/colormodel")
+  // getColorModel(): ColorModel;
```

Comment:
Disabled for now.

This op now equivalent to
```
getColorModel(): string | "Red" | "Blue" | "Green";
```
this union in response looks a bit strange (e.g. if compare to `getColorModel(): OkResponse | NoContentResponse`)

Reply:
I guess I still need to solve this now... cadl-ranch has such test case.

Reply:
Changed to explicit `@body` on Timothee's advice.